content/blogs/2024-01-16-gorgias.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Gorgias — Applying Infrastructure As Code Best Practice to Data Engineering with Kestra"
+description: "Dive into how Gorgias employs IaC principles for its data infrastructure using best-of-breed tools, including Kestra, Airbyte, dbt, Hightouch, and Terraform."
+date: 2024-01-16T09:00:00
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: "bpimpaud"
+image: /blogs/2024-01-16-gorgias.jpg
+---
+
+Infrastructure as Code (IaC) has become the industry standard for cloud resources management and configuration.
+
+It allows teams to easily collaborate across multiple resources through providers and have a common declarative syntax for several purposes: IAM policies, Kubernetes clusters, database users and tables, etc.
+
+What if this same principle were applied to Data Engineering?
+
+That's what [Gorgias](https://www.gorgias.com/) is doing with [Kestra](https://github.com/kestra-io/kestra).
+
+With the rise of the Modern Data Stack, workflows span across diverse tools — ranging from ETL scripts, data transformations, and databases to reverse-ETL processes and ML models.
+
+The current state of solutions involves scattering configurations across various domains:
+
+- **Infrastructure**: service account roles, datasets, Kubernetes clusters
+- **Data Sources** (e.g., Segment, SaaS webhooks): Subscribed events, destinations
+- **ETL tools** (Airbyte, Fivetran, custom solutions): Sources, destinations, sync frequency, retry policies
+- **Transformation** (dbt, sqlmesh, scheduled queries): commands, scheduling, retry policies, monitoring & alerting rules
```

Comment:
```suggestion
- **Transformation** (dbt, scheduled queries): commands, scheduling, retry policies, monitoring & alerting rules
```

no need to make a promo for sqlmesh here, especially because Georgias is dbt Cloud customer :)